.gitignore

```diff
@@ -145,3 +145,8 @@ __blobstorage__
 __queuestorage__
 __azurite_db*__.json
 .python_packages
+
+#summarization development
+src/summarizations/msdata_pull_request_list.csv
+src/summarizations/msazure_pull_request_list.csv
+src/summarizations/constants.py
```

Comment:
lets switch this to a configuration file like our azure.yaml

Reply:
is this one resolved?